tracing/tracingtest/mocktracer.go

```diff
@@ -0,0 +1,41 @@
+package tracingtest
+
+import (
+	"sync/atomic"
+	"time"
+
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/mocktracer"
+)
+
+type MockTracer struct {
+	*mocktracer.MockTracer
+	spans int32
+}
```

Comment:
Here we can use https://pkg.go.dev/sync/atomic#Int32 to count spans.

I think it also makes sense to add a constructor that would init *mocktracer.MockTracer to simplify caller side to:
```go
tracer := tracingtest.NewTracer()
```

Reply:
Fixed there https://github.com/zalando/skipper/pull/3322 .